app/inject/TemplateServiceProvider.java

```diff
@@ -0,0 +1,27 @@
+package inject;
+
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.FileTemplateLoader;
+import com.google.inject.Provider;
+import common.templates.HandlebarsTemplateService;
+import common.templates.TemplateService;
+import play.Logger;
+
+public class TemplateServiceProvider implements Provider<TemplateService> {
+    private static final String DEFAULT_TEMPLATE_PATH = "/META-INF/resources/webjars/templates";
+    private static final String OVERRIDE_TEMPLATE_PATH = "app/assets/templates";
+
+    @Override
+    public TemplateService get() {
+        Logger.info("execute ViewServiceProvider.get()");
```

Comment:
It was inherited code I copied it from. I'll change it in all other providers as well.